app/views/shared/_about_petitions.html.erb

```diff
@@ -5,6 +5,12 @@
   </section>
   <section class="about-item  about-item-count-100000">
     <h2>100,000 <span>signatures</span></h2>
+    <% if local_assigns[:petition] && petition.scheduled_debate_date %>
```

Comment:
This is a bit ugly. I saw that @h-lame had also touched this file in his feature PR https://github.com/alphagov/e-petitions/pull/145 - I will make this look nicer when merging these two features.